twilight-http/src/request/guild/update_guild_onboarding.rs

```diff
@@ -0,0 +1,123 @@
+use std::future::IntoFuture;
+
+use twilight_model::{
+    guild::onboarding::{Onboarding, OnboardingMode, OnboardingPromptEmoji, OnboardingPromptType},
+    id::{
+        marker::{
+            ChannelMarker, GuildMarker, OnboardingPromptMarker, OnboardingPromptOptionMarker,
+            RoleMarker,
+        },
+        Id,
+    },
+};
+
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    client::Client,
+    error::Error,
+    request::{self, AuditLogReason, Request, TryIntoRequest},
+    response::{Response, ResponseFuture},
+    routing::Route,
+};
+
+use twilight_validate::request::{audit_reason as validate_audit_reason, ValidationError};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+struct UpdateOnboardingPromptOption {
```

Comment:
maybe this should be named `OnboardingPromptOption` to follow discord docs

Reply:
consequently, this could be moved to the model crate

Reply:
The reason these are named the way they are is because they are only applicable to updating onboarding information. `GuildOnboarding…` structs in the model package represent the data received from the API.